routes/api/v1/playlists.js

```diff
@@ -111,4 +112,9 @@ router.delete('/:id', (request, response) => {
     })
 })
 
+router.use('/:playlist_id/favorites', function(request, response, next) {
+  request.playlist_id = request.params.playlist_id;
+  next()
+}, favorites);
+
```

Comment:
I think a future refactor could look into the merge params feature, but for now this works, so yay!